### PR TITLE
moved extras/jit.py -> tinygrad/jit.py

### PR DESCRIPTION
**Pull request by vovw**

### CODE REVIEW

File: examples/compile_efficientnet.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def compile_net(run, special_names):
   model = EfficientNet(0)
   model.load_from_pretrained()
 
-  from extra.jit import TinyJit
```
Comment (python273) [inline examples/compile_efficientnet.py:37]: indent

File: examples/compile_efficientnet.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def compile_net(run, special_names):
...
+from tinygrad.jit import TinyJit
```
Comment (python273) [inline examples/compile_efficientnet.py:37]: indent

Comment (vovw) [reply examples/compile_efficientnet.py]: thanks for pointing it out.

**Comment by geohot** [comment]
Tests failing

**Comment by vovw** [comment]
image
As ``tinygrad.helpers.DEBUG`` is not used should I  remove it ?

**Comment by python273** [comment]
yep

**Comment by vovw** [comment]
should pass the tests now.

**Pull request merged**